lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -607,6 +608,28 @@ class DocumentImpl extends NodeImpl {
     });
   }
 
+  // https://dom.spec.whatwg.org/#dom-document-createcdatasection
+  createCDATASection(data) {
+    if (data.indexOf("[[>") !== -1) {
```

Comment:
Copy that. Learning communication patterns for this repo @domenic . Each one is different it seems. Will update asap. Any input on the other reviews? I'm sure you know the proper suggestion in minutes. Thanks in advance.